eng/source-build-patches/0008-use-sourcebuildRID-to-find-clrjit.patch

```diff
@@ -0,0 +1,25 @@
+From 2ac4c152b5c4409025dcbaf473b89a6094453e97 Mon Sep 17 00:00:00 2001
+From: adaggarwal <aditya.aggarwal@microsoft.com>
+Date: Sat, 8 Feb 2020 06:43:48 +0000
+Subject: [PATCH 08/14] use-sourcebuildRID-to-find-clrjit
+
+---
+ .../App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj     | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj b/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj
+index 4199691fdb..6387557d8f 100644
+--- a/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj
++++ b/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj
+@@ -411,7 +411,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
+     <PropertyGroup>
+       <!-- Pick the right coreclr jit based on whether we are cross-compiling or not. -->
+       <CoreCLRJitPath
+-          Condition="'$(CrossCompileDirectory)' == ''">$(RuntimePackageRoot)runtimes\$(RuntimeIdentifier)\native\$(LibPrefix)clrjit$(LibExtension)</CoreCLRJitPath>
++          Condition="'$(CrossCompileDirectory)' == ''">$(RuntimePackageRoot)runtimes\$(SourceBuildRuntimeIdentifier)\native\$(LibPrefix)clrjit$(LibExtension)</CoreCLRJitPath>
```

Comment:
When would `(RuntimeIdentifier)` and `$(SourceBuildRuntimeIdentifier)` be different i.e. why do we need both❔

Reply:
Would appreciate an explanation here.

Reply:
I reached out to @dseefeld who has more historical context here.  He thought this was an issue in the portable build, source-build was trying to use a portable cross-gen (e.g. `RuntimeIdentifier`) and it was causing issues.